projects/common-consumption/src/lib/card/toc-card/toc-card.component.html

```diff
@@ -163,7 +166,8 @@ <h4 *ngIf="bestScoreLabel" class="sbchapter__item__score pr-8 ml-auto" title="{{
       </i>
       <img [src]="fallbackImg" width="32" height="32" *ngIf="!content?.appIcon" alt="image" #img />
     </div>
-    <div >{{content?.name}}</div>
+    <div mat-line>{{content?.relationalMetadata?.optional? "Addition Material: ":""}}{{content?.name}}</div>
```

Comment:
@Saurabhbalke  why u have added this <div mat-line>{{content?.relationalMetadata?.optional? "Addition Material: ":""}}{{content?.name}}</div> two time {{}}
why we can use like this {{content?.relationalMetadata?.optional ?  "Addition Material: " + content?.name : content?.name}}</div>

Reply:
hi @princegupta1131  
added req changes, Thanks !!